.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: ruby
+rvm:
+  - 2.4.1
+
+cache: bundler
+
+script:
+  - bundle exec rake db:setup
+  - bundle exec rake spec
+
+services:
+  - postgresql
+
+before_script:
+    - cp config/database.yml.example config/database.yml
+    - bundle exec rubocop
+    - psql -c 'create database heros_team_app_test;' -U postgres
```

Comment:
it's probably better to created DB using rails command, as manual creation may bring inconsistency in data encoding

or in this case just ensure that DB encodings match for rails app and default for `CREATE DATABASE` command

Reply:
Setup DB creating via rails command.